node/src/lib.rs

```diff
@@ -9,25 +9,21 @@ use consensus::{
 };
```

Comment:
Are we able to delete `temp_batch_store`, `temp_batch_map` and `TEMP_BATCH_CF` from this file?

Reply:
I'll take a look at that, I think right now there is still some code in block synchronizer that might use it(but yes, we should delete it)